core/src/main/java/com/zextras/carbonio/files/config/interfaces/FilesConfig.java

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package com.zextras.carbonio.files.config.interfaces;
+
+import com.zextras.carbonio.preview.PreviewClient;
+import com.zextras.carbonio.usermanagement.UserManagementClient;
+import com.zextras.filestore.api.Filestore;
+
+import java.util.Properties;
+
+public interface FilesConfig {
+  Properties getProperties();
+  UserManagementClient getUserManagementClient();
+  PreviewClient getPreviewClient();
+  Filestore getStorages();
```

Comment:
Since we are here with a mini refactor, I would change also this method name
```suggestion
  Filestore getStoragesClient();
```